react/week4/usersearch/src/components/SearchResult.js

```diff
@@ -0,0 +1,28 @@
+import React, { useContext} from "react";
+import UserContext from "./ListUserContext";
+
+function SearchResult() {
+
+const { users, loading, error } = useContext(UserContext);
+
+  if (loading) return <p className="loading">Loading...</p>;
+  if (error) return <p>{error}</p>;
```

Comment:
jus this two lines 8 and 9, if you want return HTML element I think it should be below return part of the component I mean below line 11 
but here you can implement functionality , I mean you can return values not HTML element